network-upgrades/mainnet-upgrades/shanghai.md

```diff
@@ -15,7 +15,7 @@ Changes included in the Network Upgrade.
 | Network | Timestamp  | Date & Time (UTC)  | Fork Hash    | Beacon Chain Epoch |
 |---------|------------|---------------|--------------| ---------- |
 | Sepolia | `1677557088` | 2/28/2023, 4:04:48 AM | `0xf7f9bc08` | 56832 
-| Goerli  | TBA | TBA |  TBA | TBA 
+| Goerli  | `1678857936` | 3/14/2023, 10:25:36 PM	 |  TBA | 162304 
```

Comment:
If we're aiming for 3/14/2023, 10:25:36 PM UTC, then I calculate that the timestamp should be 1678832736